File: src/test/resources/languages/java/malformed.java
package com.extractor.malformed;

public class Malformed {

    public void missingBrace() {
        if (true) {
            System.out.println("No closing brace for if");
    }

    public void unclosedString() {
        String s = "This string never ends;
    }

    public void unclosedComment() {
        /* This comment is never closed
    }

    // Unclosed class brace follows

File: src/test/resources/languages/python/malformed.py
class Malformed:
    def method_with_bad_indentation(self):
    print("This line is incorrectly indented")
        print("This one too")

def unclosed_string():
    s = "This string is never closed

def unclosed_docstring():
    """This docstring starts but never ends
    
def incomplete_def():
    def 
